static/js/admin.js

```diff
@@ -88,7 +91,8 @@ function populate_users(realm_people_data) {
     bots = _.sortBy(bots, 'full_name');
 
     _.each(bots, function (user) {
-        bots_table.append(templates.render("admin_user_list", {user: user}));
+        bots_table.append(templates.render("admin_user_list", {user: user, users_list: active_users}));
+        $("select[id='edit_bot_owner_" + user.email + "']").val(user.bot_owner || "");
```

Comment:
We're trying to move away from that style of fetching objects; I opened https://github.com/zulip/zulip/issues/3643 as a follow-up issue for this.  I think it's OK to do this feature the old way for now and plan to migrate later.